server/controller/Meetup.js

```diff
@@ -139,22 +103,19 @@ class Meetup {
          const meetup = Meetups.findIndex((meetup) => meetup.id === Number(req.params.meetupId));
 
          if (meetup === -1) {
-            return res.json({
+            return res.status(404).json({
                status: 404,
                error: `Request unsuccessful, meetup for id ${req.params.meetupId} not found`
             })
-
          } else {
             Meetups.splice(meetup, 1);
-            return res.json({
-               status: 204,
+            return res.status(200).json({
+               status: 200,
                message: `Delete Successful. Meetup with id ${req.params.meetupId} successfully deleted`
```

Comment:
consider destructuring variables especially in cases where you have to use those variables again. e.g `meetupId` above